Tests/SystemTests/ErrnoTest.swift

```diff
@@ -164,12 +164,14 @@ final class ErrnoTest: XCTestCase {
 #if !os(Windows)
     XCTAssert(EBADMSG == Errno.badMessage.rawValue)
     XCTAssert(EMULTIHOP == Errno.multiHop.rawValue)
-    XCTAssert(ENODATA == Errno.noData.rawValue)
     XCTAssert(ENOLINK == Errno.noLink.rawValue)
+    XCTAssert(EPROTO == Errno.protocolError.rawValue)
+#if !os(FreeBSD)
```

Comment:
I think it would be more straightforward to have `#if !os(Windows) ... #endif` followed by `#if !os(Windows) && !os(FreeBSD) ... #endif`.

Reply:
ya... nested ifdefs are quite gross...

Reply:
Thanks. It's better to keep a mind-numbing list as simple as possible =)